certora/specs/misc/EVC_Prop23_MustRevertFunctions.spec

```diff
@@ -0,0 +1,17 @@
+import "../utils/IsMustRevertFunction.spec";
```

Comment:
rename this file not to refer to `Prop23`

Reply:
Fixed.